client/src/components/Mapa.jsx

```diff
@@ -55,8 +55,8 @@ const Mapa = (props) =>  {
 				geom && <GeoJSON key={geomHash} data={geom} />
 			}
 			<div className="map-title">
-				{countyName && <button className="btn btn-light disabled">{countyName}</button>}
```

Comment:
this changes the way the texts in these "buttons" are styled which needs to be fixed.
plus placeholder buttons were not converted (lines 14-15)

Reply:
hmmm... I am not sure I understand your comment.
Is there any change in rendering between `<button>` and `<span>`?
I believe the style comes from `className="btn btn-light..."`
I think Lines 14-15 do not appear inside another `<button>` so they do not have issues.